analyzers/tests/SonarAnalyzer.Test/Rules/NativeMethodsShouldBeWrappedTest.cs

```diff
@@ -21,64 +21,59 @@
 using Microsoft.CodeAnalysis.CSharp;
 using SonarAnalyzer.Rules.CSharp;
 
-namespace SonarAnalyzer.Test.Rules
+namespace SonarAnalyzer.Test.Rules;
+
+[TestClass]
+public class NativeMethodsShouldBeWrappedTest
 {
-    [TestClass]
-    public class NativeMethodsShouldBeWrappedTest
-    {
-        private readonly VerifierBuilder builder = new VerifierBuilder<NativeMethodsShouldBeWrapped>();
+    private readonly VerifierBuilder builder = new VerifierBuilder<NativeMethodsShouldBeWrapped>();
 
-        [TestMethod]
-        public void NativeMethodsShouldBeWrapped() =>
-            builder.AddPaths("NativeMethodsShouldBeWrapped.cs").Verify();
+    [TestMethod]
+    public void NativeMethodsShouldBeWrapped() =>
+        builder.AddPaths("NativeMethodsShouldBeWrapped.cs").Verify();
 
 #if NET
 
-        [TestMethod]
-        public void NativeMethodsShouldBeWrapped_CSharp9() =>
-            builder.AddPaths("NativeMethodsShouldBeWrapped.CSharp9.cs").WithTopLevelStatements().Verify();
-
-        [TestMethod]
-        public void NativeMethodsShouldBeWrapped_CSharp10() =>
-            builder.AddPaths("NativeMethodsShouldBeWrapped.CSharp10.cs").WithOptions(ParseOptionsHelper.FromCSharp10).Verify();
+    [TestMethod]
+    public void NativeMethodsShouldBeWrapped_CSharp9() =>
+        builder.AddPaths("NativeMethodsShouldBeWrapped.CSharp9.cs").WithTopLevelStatements().Verify();
 
-        // NativeMethodsShouldBeWrapped.CSharp11.SourceGenerator.cs contains the code as generated by the SourceGenerator. To regenerate it:
-        // * Take the code from NativeMethodsShouldBeWrapped.CSharp11.cs
-        // * Copy it to a new .Net 7 project
-        // * Press F12 on any of the partial methods
-        // * Copy the result to NativeMethodsShouldBeWrapped.CSharp11.SourceGenerator.cs
-        [TestMethod]
-        public void NativeMethodsShouldBeWrapped_CSharp11() =>
-            builder
-                .AddPaths("NativeMethodsShouldBeWrapped.CSharp11.cs")
-                .AddPaths("NativeMethodsShouldBeWrapped.CSharp11.SourceGenerator.cs")
-                .WithOptions(ParseOptionsHelper.FromCSharp11)
-                .WithConcurrentAnalysis(false)
-                .Verify();
+    // NativeMethodsShouldBeWrapped.CSharp11.SourceGenerator.cs contains the code as generated by the SourceGenerator. To regenerate it:
```

Comment:
What happened to this test?

Reply:
It got merged with the CSharp9 test, because it does not require C#10